src/worker/sandbox/cgroups/pool.go

```diff
@@ -33,11 +33,12 @@ func NewCgroupPool(name string) (*CgroupPool, error) {
 		recycled: make(chan *CgroupImpl, CGROUP_RESERVE),
 		quit:     make(chan chan bool),
 		nextID:   0,
+		log:      common.LoadCgroupLogger("cg-pool", path.Base(path.Dir(common.Conf.Worker_dir))+"-"+name),
```

Comment:
We'll usually only have one cgroup pool, so I don't think it's worth including this path in every log line.